src/crypto/root_certs.h

```diff
@@ -0,0 +1,3365 @@
+static const char* const root_certs[] = {
```

Comment:
where are these from?

Reply:
Mozilla certdata.txt, nodejs, dart do the same we will need a update process for each release probably i will add a comment on how I generated it and create a script

Reply:
yeah this needs to be a big comment at the top explaining where the list was generated

also this is sensitive stuff from a security perspective

we probably should move our fork of uSockets to be more directly in bun's repo or at least under oven-sh

Reply:
not right now but its something to consider

Reply:
@Jarred-Sumner updated